daprdocs/content/en/developing-applications/building-blocks/actors/actor-reminders-partitions.md

```diff
@@ -0,0 +1,96 @@
+---
+type: docs
+title: "How-to: Enable multiple partitions for actor reminders in Dapr"
+linkTitle: "How-To: partition actor reminders"
+weight: 30
+description: Scale actor reminders storage via partitioning
+---
+
+{{% alert title="Preview feature" color="warning" %}}
+Actor reminders partitioning is currently in [preview]({{< ref preview-features.md >}}).
```

Comment:
Per https://github.com/dapr/docs/issues/1592 we'll move the list of current preview features to support. In this note we should link to both pages - to see all the current preview features and also to the "how to" that tells the reader how to enable it (the link currently provided here)

Reply:
Is #1592 already addressed? It is not clear what is the action item here.

Reply:
We need to add a link to the new page that is created as part of addressing #1592. We can add the link afterwards so no action right now

Reply:
ok.